components/analytics-dashboard/org.wso2.carbon.analytics.dashboard.admin/src/main/java/org/wso2/carbon/analytics/dashboard/admin/RegistryUtils.java

```diff
@@ -61,7 +62,7 @@ protected static void writeResource(String url, Object content) throws AxisFault
 		} catch (Exception re) {
 			String errorMessage = "Unable to write resource at url:" + url;
 			logger.error(errorMessage);
```

Comment:
When logging an error, pass the error object itself as well. This will print the stack trace so that will be helpfull in tracing the error.